src/main/java/org/jboss/forge/arquillian/command/AddTestFrameworkCommand.java

```diff
@@ -44,6 +48,10 @@
    @WithAttributes(shortName = 'n', label = "Test Framework Version", type = InputType.DROPDOWN)
    private UISelectOne<String> testFrameworkVersion;
 
+   @Inject
+   @WithAttributes(shortName = 's', label = "Standalone Mode", type = InputType.CHECKBOX)
+   private UIInput<Boolean> standaloneMode;
```

Comment:
I would suggest to rename it to just `standalone`. At least as the CLI flag if not the field itself.